### ISSUE TITLE
Remove Twisted Emacs to a separate project

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5205]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#5205|
|Type|enhancement|
|Created|2011-07-27 10:23:39Z|
|Branch|https://github.com/twisted/twisted/tree/remove-emacs-5205|

The top-level `emacs` directory contains neat stuff, however it's somewhat different than the rest of Twisted:

  1. Some of it is about editor customizations
  1. It's in elisp instead of Python
  1. It's poorly documented
  1. It has no unit tests

Instead of being part of Twisted proper, it should be a separate project where development can proceed more appropriately for what it is.

I've already created <http://launchpad.net/twisted-emacs>.  All we have to do is remove the `emacs` directory from the svn repository and update any documentation to refer to the new location.

<details><summary>Searchable metadata</summary>

```
trac-id__5205 5205
type__enhancement enhancement
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__branches_remove_emacs_5205 branches/remove-emacs-5205
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1311762219000000 1311762219000000
changetime__1312509937000000 1312509937000000
version__None None
owner__exarkun exarkun
cc__jesstess
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

(In [32370]) Branching to 'remove-emacs-5205'

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)|@jesstess set owner to @jesstess|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_4"></a>|@jesstess set owner to @exarkun|
|-|-|

Thanks for working on this, exarkun. Looks good, please merge.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun set status to `closed`|
|-|-|

(In [32418]) Merge remove-emacs-5205

Author: exarkun
Reviewer: jesstess
Fixes: [#5205](https://github.com/twisted/twisted/issues/5205)

Remove the emacs customization code from Twisted, in favor of maintaining it
on Launchpad as the twisted-emacs project.
